### ISSUE TITLE
ViewBox Y flipping not consistent across platforms

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/lasagna#137]
In https://github.com/raacampbell13/lasagna/commit/b4e9b5504439812165777387eaf23bce368e7619 I noticed that the Linux version requires the Y axis to be flipped but the Mac version does not. This is fixed, but I don't understand why it's necessary. Should make a stripped down example and see if even then the flipping is needed on just one platform.

**Comment by raacampbell** [comment]
I'm not certain this is true. Currently the Mac always requires the Y axis to be flipped. Lasagna always shows images upside down (flipped top/bottom). Should double-check this. 

However, we need to correct the flip because it's a real pain in the ass now that we're using Lasagna to annotate stuff. First check that flip is consistent. Then look into what to do about it.

**Comment by raacampbell** [comment]
Hernando thinks that on Linux the image stack does not appear flipped. I currently can not install on on `joiner` to confirm.

**Comment by raacampbell** [comment]
Yep: on Windows `lasagna -D` loads the stack the correct way around in the coronal plane. Only on Mac is it D/V inverted.

**Comment by raacampbell** [comment]
I see. In `lasagna_viewBox` I find:

```
# TODO: why the hell does the Mac version not require the flip but the Linux version does. What about Win?
if platform.system() != 'Darwin':
    self.invertY()
```

Commenting out this code unsurprisingly removes the flip on Mac. After this, it's possible to trace stuff, load the trace in MATLAB, and overlay with no problem.

Fixed with: a24dc9ffb72ac247c8770ef608fb526beb5d4d16

**Comment by raacampbell** [comment]
Should be flipped on **ALL** platforms f6cca80ef4985886a840cc3bdc4c0ee6ae1fc64a
